packages/electrode-archetype-react-app/lib/features.js

```diff
@@ -62,27 +40,38 @@ class Feature {
   }
 
   async attachNpmAttributes() {
-    const { description, electrodeOptArchetype, version } = await this.getNpmAttributes([
-      "description",
-      "electrodeOptArchetype",
-      "version"
-    ]);
+    const { description, electrodeOptArchetype, version } = await this.getNpmAttributes();
     this.npmDescription = description;
     this.npmElectrodeOptArchetype = electrodeOptArchetype;
     this.npmVersion = version;
   }
 
-  async getNpmAttributes(fields) {
-    const { stdout } = await exec(`npm view ${this.packageName} ${fields.join(" ")}`);
-    const regex = fields.map(field => `${field} =(.*)`).join("");
-    const values = stdout
-      .replace(/[\n\r]/g, "") // No newlines
-      .match(regex)
-      .splice(1)
-      .map(parseRelaxedJson);
-    const fieldMap = {};
-    fields.forEach((field, index) => (fieldMap[field] = values[index]));
-    return fieldMap;
+  async getNpmAttributes() {
+    const pathName = `.etmp/${this.packageName}.registry.json`;
```

Comment:
1. The Electrode Tmp dir is available from `appArchetypeConfig.eTmpDir`
2. It's probably safer to do a `Path.resolve` to get the full path.